packages/json-schema/src/json-schema-emitter.ts

```diff
@@ -344,6 +344,10 @@ export class JsonSchemaEmitter extends TypeEmitter<Record<string, any>, JSONSche
           schema = { type: "integer" };
         }
         break;
+      case "decimal":
```

Comment:
So we want to do string here not number like we ended up doing for openapi?

Reply:
More to the point, I think this should follow the int64-strategy which uses string by default. Can add an option in a follow-on PR to emit numbers for this instead.